ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -253,12 +269,18 @@ def install_internal_cluster(self):
         logger.info("Select all worker nodes")
         self.select_checkbox_status(status=True, locator=self.dep_loc["all_nodes"])
 
+        logger.info("Enable Infra Nodes")
+        if self.ocp_version == "4.10" and config.DEPLOYMENT.get("infra_nodes", False):
+            self.select_checkbox_status(
+                status=True, locator=self.dep_loc["enable_infra_node"]
+            )
+
         if self.ocp_version == "4.6" and config.ENV_DATA.get("encryption_at_rest"):
             self.do_click(
                 locator=self.dep_loc["enable_encryption"], enable_screenshot=True
             )
 
-        if self.ocp_version in ("4.7", "4.8", "4.9"):
+        if self.ocp_version in ("4.7", "4.8", "4.9", "4.10"):
```

Comment:
Can't we condition this line as-
```
        if self.ocp_version >= "4.7":
```

Reply:
No, for example:
```
"4.10">"4.7"
False
```
I can use get_semantic_ocp_version_from_config() method